lldb/scripts/dpu/dpu_commands.py

```diff
@@ -50,9 +51,9 @@ def get_dpu_from_command(command, debugger, target):
                              str(dpu[1]) + "." + str(dpu[2]) + "." +
                              str(dpu[3]) + "." + str(dpu[4]))),
                         0)
-            if addr == 0:
-                print("Could not interpret command '" + command + "'")
-                return None
+    if addr == 0:
+        print("Could not interpret command '" + command + "'")
+        return None
```

Comment:
The indentation was not a mistake. Did you encounter a issue with it?

Reply:
Yes, because of the change in `get_value_from_command`.